README.md

```diff
@@ -52,6 +52,17 @@ Tracks the commits in a [git](http://git-scm.com/) repository.
   the `branch`. Patterns are [glob(7)](http://man7.org/linux/man-pages/man7/glob.7.html)
   compatible (as in, bash compatible).
 
+* `submodules`: *Optional.* List of credentials for HTTP(s) auth when pulling/pushing private git submodules which are not stored in the same git server as the container repository.
```

Comment:
Can we call this `submodule_credentials` or something instead? I could see it being confused with the `submodules` param on the `get` step.

Reply:
That's a good catch! I have corrected it in the latest commit